scripts/load-development-data

```diff
@@ -43,9 +50,11 @@ function load_database_backup() {
 
     echo "Restore database from backup"
     # Command to create database backup
-    # gosu postgres pg_dump -Fc granary > database.pgdump
+    # docker-compose \
```

Comment:
:wastebasket:

Reply:
Ah, I was actually going to keep this around since it documents how to create it